packages/create-app/templates/default-app/docker-compose.yaml.hbs

```diff
@@ -0,0 +1,44 @@
+version: "3.4"
+services:
+  {{#if dbTypePG}}
+  postgres:
+    image: postgres:11-alpine
+    container_name: postgres
+    restart: always
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: backstage
+      POSTGRES_PASSWORD: backstage
+    volumes:
+      - ./tools/development/db/:/docker-entrypoint-initdb.d
+  {{/if}}
+  backstage:
```

Comment:
Not sure about making this compose script a part of the standard template tbh. It's much simpler and less resource intense to just use SQLite for local development.

Perhaps this makes sense to put in `/contrib`, as suggested by #2459?

Reply:
I agree it's simpler from bootstrap, but I would argue it isn't obvious how to upgrade to a more production like environment. The CLI doesn't hint that for your db options, you're really choosing between a local vs production setup. A dev which selects Postgres (since that's what will be running in prod), would be left with a broken development environment.

Reply:
So the contrib stuff has landed, and I think it's best to put this there now. The tricky bit about deployment is that there are so many different strategies and tools, making it tricky to standardize on any one way of doing things. We provide a way to build docker images, but that's really where out opinionated way of doing things end and the community gets to take over. The `/contrib` folder is intended to contain many of the different strategies that can be used to deploy Backstage, among other things.

Reply:
Agree regarding Postgres btw, and tbh I think we should remove it as an option in `create-app`

Reply:
Ah right that makes sense 🙂 I'll move this into `contrib`. Should we remove the postgres option in this PR?

Reply:
Probably do that in a followup so we can have a focused discussion around that. If it's any kind of effort I'd start with an issue to discuss it first too though, or a draft PR that doesn't cover everything.